packages/web/src/components/premium-content-purchase-modal/components/PurchaseContentFormFields.tsx

```diff
@@ -19,64 +35,122 @@ import { PurchaseSummaryTable } from './PurchaseSummaryTable'
 
 const messages = {
   purchaseSuccessful: 'Your Purchase Was Successful!',
-  manualTransfer: '(Advanced) Manual Crypto Transfer'
+  existingBalance: 'Existing balance',
+  card: 'Add funds with card',
+  manualTransfer: 'Add with crypto transfer',
+  paymentMethod: 'Payment Method',
+  dollarSign: '$'
 }
 
 type PurchaseContentFormFieldsProps = Pick<
   PurchaseContentFormState,
   'purchaseSummaryValues' | 'stage' | 'isUnlocking'
->
+> & { price: number }
 
 export const PurchaseContentFormFields = ({
+  price,
   purchaseSummaryValues,
   stage,
   isUnlocking
 }: PurchaseContentFormFieldsProps) => {
-  const { onOpen: openUsdcManualTransferModal } = useUSDCManualTransferModal()
+  const { data: balance } = useUSDCBalance()
   const payExtraAmountPresetValues = usePayExtraPresets(useRemoteVar)
+  const [{ value: purchaseMethod }, , { setValue: setPurchaseMethod }] =
+    useField(PURCHASE_METHOD)
   const isPurchased = stage === PurchaseContentStage.FINISH
+  const balanceUSDC = USDC((balance ?? new BN(0)) as BN)
+  const { extraAmount } = usePurchaseSummaryValues({
+    price,
+    currentBalance: balance
+  })
+  const isExistingBalanceDisabled =
+    USDC(price / 100).value + USDC((extraAmount ?? 0) / 100).value >
```

Comment:
maybe we should have a `CENTS()` fixed decimal constructor helper, so this could be
`USDC(CENTS(price))`. Basically a helper for `USDC(new FixedDecimal(BigInt(price), 2))`. Although, seems like we should be converting `price` to USDC earlier in the stack as I see this conversion happening a lot...

nothing blocking here, don't need to change, but thoughts?